setup.py

```diff
@@ -13,7 +13,10 @@
     long_description = f.read()
 
 install_reqs = parse_requirements('requirements.txt', session='hack')
```

Comment:
It seems that using `pip._internal.req` is a not a good practice (the module can change at every new version of pip). In general, parsing the requirements file to fill the file `setup.py` is not adviced (https://packaging.python.org/discussions/install-requires-vs-requirements/). We need to refactor the packaging process in the next release.